fuel-tx/src/transaction/consensus_parameters.rs

```diff
@@ -251,9 +251,10 @@ impl TxParameters {
     /// Transaction memory offset in VM runtime
     pub const fn tx_offset(&self) -> usize {
         Bytes32::LEN // Tx ID
-            + WORD_SIZE // Tx size
-            // Asset ID/Balance coin input pairs
-            + self.max_inputs as usize * (AssetId::LEN + WORD_SIZE)
+        + AssetId::LEN // Base asset ID
+        // Asset ID/Balance coin input pairs
+        + self.max_inputs as usize * (AssetId::LEN + WORD_SIZE)
+        + WORD_SIZE // Tx size
```

Comment:
Why is the `Tx size` spot flipped from before?

Reply:
It's after the balance pairs in memory, so I made the order here match the order in memory.

Reply:
Ah. It looks like it wasn't matching the specs before but it is now AFAICT.